fairscale/experimental/wgit/pygit.py

```diff
@@ -123,9 +123,14 @@ def _path(self) -> str:
         """returns the path of the git repository PyGit is wrapped around"""
         return self.repo.path
 
-    def status(self) -> None:
+    def status(self) -> Dict:
         """Show the status of the git repo"""
-        print(self.repo.status())
+        status_dict = self.repo.status()
+        tracking_dict = {}
+        for key, val in status_dict.items():
+            if val != pygit2.GIT_STATUS_IGNORED:
+                tracking_dict[key] = val
+        return tracking_dict
```

Comment:
these should be able to be reduce to a single filter() line to the filter the status_dict, right? Like:

return filter(status_dict, ...)